base/bucket.go

```diff
@@ -111,6 +109,7 @@ type BucketSpec struct {
 	Auth                          AuthHandler
 	CouchbaseDriver               CouchbaseDriver
 	Certpath, Keypath, CACertPath string         // X.509 auth parameters
+	CACertUnsetTlsSkipVerify      bool           // Use insecureSkipVerify when secure scheme (couchbases) is used and cacertpath is undefined
```

Comment:
I think the simpler name TlsSkipVerify would make sense here, and (as mentioned below), only set this when CACertPath isn't specified in the config.  That way we don't need to do the "should I use CACertPath or TLSSkipVerify" every time we use the spec.